### PR TITLE
Feature 7/attach detach schema, send notification

### PR DESCRIPTION
**Pull request by elchinmemphis**
PR intended to resolve [Attach/Detach schema to/from station](https://github.com/memphisdev/memphis.net/issues/7)

### CODE REVIEW

File: examples/Station/Program.cs [hunk 1]
```diff
@@ -31,6 +31,14 @@ public static async Task Main(string[] args)
                     });
 
                 Console.WriteLine("Station created successfully...");
+
+                await client.AttachSchema(stationName: station.Name, schemaName: "test-schema-01");
```
Comment (idanasulinmemphis) [inline examples/Station/Program.cs:35]: example files shouldn't include every feature of the SDK so please stick to the examples of the other SDKS

Comment (elchinmemphis) [reply examples/Station/Program.cs]: Got it. Fixed

**Comment by elchinmemphis** [comment]
The below [issue](https://github.com/memphisdev/memphis.net/issues/5) also covered in this PR

**Comment by elchinmemphis** [comment]
The mentioned issues are resolved in the scope of this PR,
@idanasulinmemphis Could you please review and merge ?

[Feature: Attach/Detach schema to/from station #7
](https://github.com/memphisdev/memphis.net/issues/7)
[Feature: Send notification to memphis-broker, when schema validation failed on producer side #5
](https://github.com/memphisdev/memphis.net/issues/5)

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**